.github/dependabot.yml 

```diff
@@ -0,0 +1,14 @@
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    target-branch: "main"
+    allow:
+      - dependency-name: "@metamask/*"
+        dependency-type: "development"
```

Comment:
Comment from @Gudahtt: `I'm curious why this is here. I would think we'd want all updates to our packages, not just those that are devDependencies. 🤔`